New f: 1
New f: 2
New f: 3
New f: 4
New f: 5
Clearing cache... (4.60715)
Cache cleared.
Clearing cache... (6.26424)
Cache cleared.
Clearing cache... (6.40104)
Cache cleared.
Clearing cache... (6.9571)
Cache cleared.
Clearing cache... (6.95739)
Cache cleared.
New f: 6
Clearing cache... (7.36004)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.36113)
Cache cleared.
Clearing cache... (7.41276)
Cache cleared.
Clearing cache... (7.95138)
Cache cleared.
terminate called after throwing an instance of 'std::bad_alloc'
  what():  std::bad_alloc
